linux/buildScript.sh

```diff
@@ -11,12 +11,12 @@ set -u
 
 srctgz="https://www.census.gov/ts/x13as/unix/x13ashtmlsrc_V1.1_B19.tar.gz"
 file=$(basename ${srctgz})
-
 td=$(tempfile -p x13)
+rm -r $td
```

Comment:
Is there a reason this needs to run as root? I assumed it wouldn't. The mktemp approach in the new script looks better overall, so I'd change it to mirror that. What is your strategy for builds in this repo? Do we want binaries for the old and new versions here?